node/core/av-store/src/lib.rs

```diff
@@ -57,34 +57,31 @@ mod columns {
 #[derive(Debug, Error)]
 enum Error {
 	#[error(transparent)]
-	ChainAPI(#[from] ChainApiError),
+	RuntimeApi(#[from] RuntimeApiError),
+	#[error(transparent)]
+	ChainApi(#[from] ChainApiError),
 	#[error(transparent)]
 	Erasure(#[from] erasure::Error),
 	#[error(transparent)]
 	Io(#[from] io::Error),
 	#[error(transparent)]
-	ChainApiChannelIsClosed(#[from] oneshot::Canceled),
```

Comment:
wait, did we merge these changes as part of #1864? :sweat_smile:

Reply:
Yeah :( I thought we would wait until this is merged, but I required some up-to-date code to see where it breaks.